docs/contributing/contributing.md

```diff
@@ -0,0 +1,71 @@
+# Contributing
+
+Thank you for considering contributing to FTSnext! We welcome your feedback, bug reports, and
+enhancement suggestions. This document outlines how to contribute to our project.
+
+## How to Obtain the Software
+
+Our project is hosted on GitHub. You can clone the repository using the following command:
+
+```bash
+git clone https://github.com/medizininformatik-initiative/fts-next.git
+```
+
+## Providing Feedback
+
+### Bug Reports and Enhancement Suggestions
+
+To provide feedback, report bugs, or suggest enhancements, please open an issue on the GitHub
+[issue][issues] page.
+
+### Discussions
+
+For general discussions or questions, use the GitHub [Discussions][discussions] tab. This is
+a great place to engage with the community and get help or provide feedback.
+
+## Contributing to the Software
+
+We welcome contributions via pull requests (PRs). Here's how you can contribute:
+
+1. **Fork the Repository**: Create your own copy of the repository by forking it on GitHub.
+2. **Create a Branch**: Create a new branch for your feature or bug fix.
+3. **Implement Your Changes**: Make your changes following our coding standards (see below).
+4. **Open a Pull Request**: Once your changes are ready, open a PR. Ensure that your PR corresponds
+   to an existing issue that documents what needs to be implemented.
+
+*Note that each pull request should have a corresponding issue discussing the changes from a
+functional perspective, while discussions in the PR comments should regard implementation
+details, code and code quality.*
+
+### Requirements for Acceptable Contributions
+
+* **Coding Standard**: All code should be formatted using `google-java-format`. PRs will be checked
+  for formatting errors.
+* **Code Quality**: CodeQL will check for known errors in the code. Ensure your code passes these
+  checks.
+* **Testing**: New committed code must be tested. Code coverage will be collected automatically, and
+  the patch diff must be 100%. Maintainers may skip these requirements if necessary.
+* **Code Readability**: Use comments to indicate hard-to-understand code snippets, but aim to name
+  all entities so that the code is as human-readable as possible.
+
+### Build Tools
+
+We use Maven and makefiles as build tools. Ensure your development environment is set up to use
+these tools.
+
+## Test Policy
+
+Our test policy requires that all new code is tested. Code coverage will be collected automatically,
```

Comment:
This is a duplicate of `### Requirements for Acceptable Contributions`

Can we combine them?